parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -251,13 +274,18 @@ installKubeletKubectlAndKubeProxy() {
         # kube binaries used by AKS and Kubernetes upstream.
         # TODO(mainred): let's see if necessary to auto-detect the path of kubelet
         logs_to_events "AKS.CSE.installKubeletKubectlAndKubeProxy.extractKubeBinaries" extractKubeBinaries ${KUBERNETES_VERSION} ${CUSTOM_KUBE_BINARY_DOWNLOAD_URL}
+        install_default_if_missing=false
+    elif [[ ! -z ${PRIVATE_KUBE_BINARY_DOWNLOAD_URL} ]]; then
+        # remove the kubelet binaries to make sure the only binary left is from the PRIVATE_KUBE_BINARY_DOWNLOAD_URL
```

Comment:
this comment is a bit misleading since we aren't actually ever going to download from PRIVATE_KUBE_BINARY_DOWNLOAD_URL according to the current implementation of extractPrivateKubeBinaries

Reply:
updated, thanks